internal/pally/opts.go

```diff
@@ -83,3 +85,53 @@ func (o Opts) copyLabels() map[string]string {
 	}
 	return l
 }
+
+// LatencyOpts configure Latencies and LatenciesVectors.
+type LatencyOpts struct {
+	Opts
+
+	Unit    time.Duration
+	Buckets []time.Duration // upper bounds
```

Comment:
Documented somewhat, but this will probably be clarified best by examples down the road.